image_analysis/timetable_analysis.py

```
@@ -482,46 +517,23 @@ async def upload_timetable(request: Request, file: UploadFile = File(...)):
-            await asyncio.wait_for(_job_queue.put((job_id, img)), timeout=30.0)
+            await asyncio.wait_for(_job_queue.put((job_id, user_id, img, token)), timeout=10.0)
         except asyncio.TimeoutError:
             _job_store.pop(job_id, None)
-            return JSONResponse(status_code=503, content={"error": "server is busy"})
+            _active_users.discard(user_id)
+            return JSONResponse(status_code=504, content={"error": "Server is busy"})
 
-        return JSONResponse(status_code=202, content={"job_id": job_id})
+        return JSONResponse(status_code=204, content={"job_id": job_id})
 
     except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)